packages/core/src/commands/network-generate.ts

```diff
@@ -111,55 +146,72 @@ export class Command extends Commands.Command {
+        { name: "feeStaticHtlcClaim", description: "Fee for HTLC claim transactions.", schema: Joi.number(), default: 0 },
+        { name: "feeStaticHtlcRefund", description: "Fee for HTLC refund transactions.", schema: Joi.number(), default: 0 },
+
+        // Dynamic fee
+        { name: "feeDynamicEnabled", description: "Dynamic fee enabled", schema: Joi.boolean() },
+        { name: "feeDynamicMinFeePool", description: "Minimum dynamic fee to enter the pool.", schema: Joi.number() },
+        { name: "feeDynamicMinFeeBroadcast", description: "Minimum dynamic fee to broadcast.", schema: Joi.number() },
+        { name: "feeDynamicBytesTransfer", description: "Dynamic fee for transfer transactions.", schema: Joi.number() },
+        { name: "feeDynamicBytesSecondSignature", description: "Dynamic fee for second signature transactions.", schema: Joi.number() },
+        { name: "feeDynamicBytesDelegateRegistration", description: "Dynamic fee for delegate registration transactions.", schema: Joi.number() },
+        { name: "feeDynamicBytesVote", description: "Dynamic fee for vote transactions.", schema: Joi.number() },
+        { name: "feeDynamicBytesMultiSignature", description: "Dynamic fee for multi signature transactions.", schema: Joi.number() },
+        { name: "feeDynamicBytesIpfs", description: "Dynamic fee for IPFS transactions.", schema: Joi.number() },
+        { name: "feeDynamicBytesMultiPayment", description: "Dynamic fee for multi payment transactions.", schema: Joi.number() },
+        { name: "feeDynamicBytesDelegateResignation", description: "Dynamic fee for delegate registration transactions.", schema: Joi.number() },
+        { name: "feeDynamicBytesHtlcLock", description: "Dynamic fee for HTLC lock transactions.", schema: Joi.number() },
+        { name: "feeDynamicBytesHtlcClaim", description: "Dynamic fee for HTLC claim transactions.", schema: Joi.number() },
+        { name: "feeDynamicBytesHtlcRefund", description: "Dynamic fee for HTLC refund transactions.", schema: Joi.number() },
 
         // Env
-        { name: "coreDBHost", description: "Core database host.", schema: Joi.string(), required: false, promptType: "", default: "localhost" },
-        { name: "coreDBPort", description: "Core database port.", schema: Joi.number(), required: false, promptType: "", default: "5432" },
-        { name: "coreDBUsername", description: "Core database username.", schema: Joi.string(), required: false, promptType: "" },
-        { name: "coreDBPassword", description: "Core database password.", schema: Joi.string(), required: false, promptType: "" },
-        { name: "coreDBDatabase", description: "Core database database.", schema: Joi.string(), required: false, promptType: "" },
+        { name: "coreDBHost", description: "Core database host.", schema: Joi.string(), default: "localhost" },
+        { name: "coreDBPort", description: "Core database port.", schema: Joi.number(), default: "5432" },
```

Comment:
```suggestion
        { name: "coreDBPort", description: "Core database port.", schema: Joi.number(), default: 5432 },
```
Just to keep coherency between schema validation `number()` and default value.